apps/courses/src/routes/index.tsx

```diff
@@ -24,54 +29,192 @@ const UnauthRoute = (props) => {
   return !user ? <Route {...props} /> : <Navigate to="/users/dashboard" />;
 };
 
+const RouteWrapper = ({
+  noHeader = false,
+  noFooter = false,
+  blackHeader = false,
+  inConcept = false,
+  children,
+}) => (
+  <>
+    {!noHeader && <Header noScrolling={blackHeader} />}
+    <Box
+      minHeight="100vh"
+      display="grid"
+      gridTemplateRows={!inConcept ? '1fr' : '1fr auto'}
```

Comment:
We can simply say `noFooter ? '1fr' : '1fr auto'` here. There's no need for `inConcept` anymore since it is redundant to `noFooter`. There will never be a footer in the concept. :)